python/src/server.py

```diff
@@ -0,0 +1,65 @@
+from konan_sdk.konan_service.services import KonanService
```

Comment:
Can we add another module called example, that has a full implementation?  I think it will give more clarity to the user, what do you think?

Reply:
This is the plan, but I'd rather postpone it a bit till we have more meaningful examples to show, and in the mean time, the sample `server.py` file is clear enough, especially alongside the `konan-docs`